internal/k8s/registry.go

```diff
@@ -94,7 +94,11 @@ func (r *registryAsync) Registry(ctx context.Context) container.Registry {
 		}
 
 		portSpec := portSpecs[0]
-		r.registry = container.NewRegistry(fmt.Sprintf("localhost:%d", portSpec.NodePort))
+		host := fmt.Sprintf("localhost:%d", portSpec.NodePort)
+		r.registry, err = container.NewRegistry(host)
```

Comment:
this sets `r.registry` if there's an error, which i'm pretty sure will cause problems?

Reply:
good catch